worlds/bk_sudoku/docs/en_Sudoku.md

```diff
@@ -0,0 +1,9 @@
+# Bk Sudoku
+
+## What is this game?
+
+Bk Sudoku is not like your avarage Archipelago game, it is instead a generic Sudoku client that can connect to an existing multiworld. When connected to an MultiWorld you can play the game to unlock random hints for your game. While slow and not very effective it will give you something to do when you have no checks left in your game.
+
+## Where is the settings page?
+
+There is no settings page, this game cannot be used in your yamls, however the client can connect to any other generated slot
```

Comment:
While technically true I'd prefer you put in a way for the settings page link to not exist for this game rather than leading to a 503 error
![Screenshot_13](https://user-images.githubusercontent.com/13184667/184723996-e56e3e08-ef4f-4fca-bd15-1dcf14d53553.png)
.

Reply:
its not leading to a 503 error when its releasesd

Reply:
well it leads to a "this page is out of logic" page since the site has 503 handling

Reply:
hmmz `settings_page = "https://archipelago.gg/games/Sudoku/info/en"` is seposed to make the settings link back to game page where they can read there are no settings

Reply:
Ah ok it does work it's just because you hardcoded the link, i see. A relative link would probably be better since it's going to stay on the site and that way it works for external webhosts too.

Reply:
I am not sure if the field supports relative links but I can try